app/utils/numbers.ts

```diff
@@ -20,3 +20,23 @@ export function callableUSPhoneNumber(phoneNumber: string) {
   const formattedNumber = `+1${numbersOnly}`;
   return formattedNumber;
 }
+
+/**
+ * Formats a phone number string into the format XXX-XXX-XXXX.
+ *
+ * E.g.
+ * "1234567890" -> "123-456-7890"
+ */
+export const formatPhoneNumber = (number: string) => {
```

Comment:
I think this type signature needs to accommodate for falsy input and is missing a return type?

```suggestion
export const formatPhoneNumber = (number: string | null | undefined): string => {
```